WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -18,6 +18,9 @@ extension WPStyleGuide {
                      .foregroundColor: sectionHeaderTextColor]
         }
 
+        // ListTableViewCell
+        public static let unreadIndicatorColor = UIColor.muriel(color: .init(name: .blue, shade: .shade30))
```

Comment:
This can just be `UIColor.primaryLight`.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/16885/commits/c2e5665e291b16065b9edb5d2fba013277428dde. Thanks!